pkg/controller/controller_binding_test.go

```diff
@@ -889,9 +888,6 @@ func TestReconcileServiceInstanceCredentialDeleteFailedServiceInstanceCredential
 	binding.ObjectMeta.DeletionTimestamp = &metav1.Time{}
 	binding.ObjectMeta.Finalizers = []string{v1alpha1.FinalizerServiceCatalog}
 
-	checksum := checksum.ServiceInstanceCredentialSpecChecksum(binding.Spec)
```

Comment:
Don't you need to set generation and reconciled generation here, or are they both zero?  I would prefer to explicitly set them both to one.